android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -443,6 +476,21 @@ fun getPromotionalOffer() : ErrorContainer {
 
 // region private functions
 
+internal class InvalidProrationMode(): Exception()
+
+@Throws(InvalidProrationMode::class)
+internal inline fun getGoogleProrationMode(oldProductId: String?, prorationMode: Int?) : GoogleProrationMode?  {
+    return prorationMode
+        ?.takeIf { oldProductId != null }
```

Comment:
Hmm why do we need this? If from the hybrids we pass an empty `oldProductId` but a proration mode, seems like that would be an issue in the hybrids. I think phc just needs to map the `prorationMode` int to the enum?

Reply:
It was just for an additional safety check 🤷‍♂️ Didn't want to possibly throw an exception if a bad proration mode was entered for some reason. Want to limit the check _only_ if we really needed to (which is when an `oldProductId` is also set.

But probably overkill 😅  I'm fine removing this if you think that's better! 

Thoughts?